internal/miners.json

```
@@ -127,6 +127,34 @@
                 "pool": "-o",
                 "pass": "-p"
             }
+        },  "GMiner": {
```

```suggestion
        },  
        "GMiner": {
```
absolutely horribly written would not ercommend

this is how you do it right